src/components/DragAndDrop/DropZoneViewHolder.js

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import {PortalHost} from '@gorhom/portal';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+    /** Name which will be used to associate view holder with its dropzone */
+    name: PropTypes.string.isRequired,
+};
+
+const DropZoneViewHolder = props => (
```

Comment:
Why do we need this component?

Reply:
So we can exchange portal implementation without changing the client code

Reply:
Sorry, I didn't quite catch that.

Reply:
I would argue against this since DropZoneViewHolder also introduces proper lingo for connecting the ids between views and is self documenting on how the system works.

However, i can revert the change - no problem.